### PR TITLE
Implemented users list and users list item components

### PR DESCRIPTION
**Pull request by adragicevicc**

### CODE REVIEW

File: src/components/RentalsOverview/RentalsOverview.tsx [hunk 1]
```diff
@@ -39,10 +42,22 @@ const RentalsOverview = ({ componentType }: RentalsOverviewProps) => {
...
+      >
         <span>Book</span>
         <span>Start date</span>
         <span>End date</span>
+        {isRoleAdmin() ? (
+          <>
+            <span>User email</span>
+            <span>Extend rent period for (days):</span>
+          </>
+        ) : null}
```
Comment (vkotarlic) [inline src/components/RentalsOverview/RentalsOverview.tsx:60]: This part should be with `&&` as I suggested, guess it was lost during rebase :)

File: src/components/RentalsOverviewItem/RentalsOverviewItem.tsx [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+            <input
+              className={classes['c-rentals-overview-item__input']}
+              name="rentPeriod"
+              onChange={handleRentPeriod}
+              type="number"
+            ></input>
+            <Button content={'Extend rent'} onClick={extendRent} disabled={!additionalRentPeriod} />
+            <Button content={'Close rent'} onClick={closeRent} />
+          </>
+        ) : null}
```
Comment (vkotarlic) [inline src/components/RentalsOverviewItem/RentalsOverviewItem.tsx:53]: This is same as comment above

File: src/services/rentalsService.ts [hunk 3]
```diff
@@ -8,19 +8,47 @@ interface RentalsParams {
...
+
+export const updateExtendRental = async (id: number, additionalRentPeriod: number) => {
+  const response = await axios.put(
+    `rent/${id}`,
+    {},
+    {
+      params: { additionalRentPeriod: additionalRentPeriod },
+      headers: {
+        Authorization: `Bearer ${localStorage.getItem('token')}`
+      }
```
Comment (vkotarlic) [inline src/services/rentalsService.ts:36]: Here headers object should be removed as we set up interceptor

File: src/services/rentalsService.ts [hunk 4]
```diff
@@ -8,19 +8,47 @@ interface RentalsParams {
...
+};
+
+export const updateCloseRental = async (id: number) => {
+  const response = await axios.put(
+    `rent/close/${id}`,
+    {},
+    {
+      headers: {
+        Authorization: `Bearer ${localStorage.getItem('token')}`
+      }
```
Comment (vkotarlic) [inline src/services/rentalsService.ts:50]: Here headers object should be removed as we set up interceptor

File: src/services/rentalsService.ts [hunk 5]
```diff
@@ -8,19 +8,47 @@ interface RentalsParams {
...
+    }
+  );
+  console.log('sada ', response);
+  return response.data;
+};
+
+export const updateCloseRental = async (id: number) => {
+  const response = await axios.put(
+    `rent/close/${id}`,
+    {},
```
Comment (vkotarlic) [inline src/services/rentalsService.ts:46]: Since we will remove config object we don't need this one

**Review by vkotarlic — CHANGES_REQUESTED** [review]

File: src/components/UsersList/UsersList.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+import UserModel from 'models/UserModel';
+import { getUsers } from 'services/userService';
+import classes from './UsersList.module.scss';
+
+interface Pagable {
+  current?: boolean;
+  page: number;
+  size: number;
+  sort: 'asc' | 'desc';
+}
```
Comment (petarnqode) [inline src/components/UsersList/UsersList.tsx:12]: Where do we use this interface, is it common?

File: src/components/UsersList/UsersList.tsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      <div className={classes['c-users-list__list-container']}>
+        {users.map((item) => (
+          <UsersListItem
+            key={item.id}
+            id={item.id}
+            email={item.email}
+            firstName={item.firstName}
+            lastName={item.lastName}
+            address={item.address}
+            phoneNumber={item.phoneNumber}
```
Comment (petarnqode) [inline src/components/UsersList/UsersList.tsx:44]: This can get messy, can we send object then destruct inside this component or {...item}

**Review by petarnqode — CHANGES_REQUESTED** [review]

**Pull request merged**